src/maxdiffusion/input_pipeline/input_pipeline_interface.py

```diff
@@ -98,7 +98,6 @@ def create_dict(moments, input_ids):
       .map(prepare_sample, num_parallel_calls=AUTOTUNE)
       .map(lambda x, y: tf.py_function(partial_tokenize, inp=[x, y], Tout=(tf.float32, tf.float32)), num_parallel_calls=AUTOTUNE)
       .map(create_dict, num_parallel_calls=AUTOTUNE)
-      .shuffle(global_batch_size * 10, seed=config.seed)
       .batch(global_batch_size // jax.process_count(), drop_remainder=False)
```

Comment:
This might still affect convergence, I got worse result. maybe add it at later PR?